lib/csv.rb

```diff
@@ -1005,7 +1005,7 @@ class << self
     def instance(data = $stdout, **options)
       # create a _signature_ for this method call, data object and options
       sig = [data.object_id] +
-            options.values_at(*DEFAULT_OPTIONS.keys.sort_by { |sym| sym.to_s })
+            options.values_at(*DEFAULT_OPTIONS.keys.sort!)
```

Comment:
We can remove sorting here because we don't support old Ruby that doesn't use inserted-order for `Hash`:

```suggestion
            options.values_at(*DEFAULT_OPTIONS.keys)
```